ChangeLog.d/conditionalize-mbedtls_mpi_sub_abs-memcpy.txt

```diff
@@ -0,0 +1,5 @@
+Bugfix
+   * Fix mbedtls_mpi_sub_abs() to account for the possibility that the output
+     pointer could equal the first input pointer and if so to skip a memcpy()
+     call that would be redundant.  Reported by Pascal Cuoq using TrustInSoft
+     Analyzer in #6701; observed independently by Aaron Ucko under Valgrind.
```

Comment:
I think the first sentence could be simply

`Fix potential undefined behaviour in mbedtls_mpi_sub_abs ().`

(ChangeLog entries are user facing, and users probably don't need to know that level of detail)